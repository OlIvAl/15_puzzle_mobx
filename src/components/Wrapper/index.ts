import styled from 'styled-components';

const StyledWrapper = styled.div`
  display: flex;
  align-items: center;
  flex-direction: column;
`;

export default StyledWrapper;