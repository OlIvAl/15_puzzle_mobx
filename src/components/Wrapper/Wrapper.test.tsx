import * as React from 'react';
import Wrapper from '.';
import renderer from 'react-test-renderer';

describe('Wrapper', () => {
  it('он отображается', () => {
    const tree = renderer.create(<Wrapper />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});