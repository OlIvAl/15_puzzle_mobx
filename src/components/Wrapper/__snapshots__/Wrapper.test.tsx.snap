// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper он отображается 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div
  className="c0"
/>
`;
